File: test/fixtures/kotlin/j_post_data_wo_verb.kt
import java.io.IOException
import okhttp3.FormBody
import okhttp3.OkHttpClient
import okhttp3.Request

val client = OkHttpClient()

val formBody = FormBody.Builder()
  .add("data1", "data1")
  .add("data2", "data2")
  .add("data3", "data3")
  .build()

val request = Request.Builder()
  .url("http://localhost:28139/post")
  .post(formBody)
  .header("Content-Type", "application/x-www-form-urlencoded")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}

File: test/fixtures/r/j_post_data_wo_verb.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = list(
  data1 = "data1",
  data2 = "data2",
  data3 = "data3"
)

res <- httr::POST(url = "http://localhost:28139/post", httr::add_headers(.headers=headers), body = data, encode = "form")
